modules/social_features/social_featured_content/templates/node--featured.html.twig

```diff
@@ -152,10 +155,10 @@
     {% block card_actionbar %}
 
       {% if visibility_icon and visibility_label %}
-        <div class="badge teaser__badge no-padding"
-             title="{% trans %}The visibility of this content is set to {{ visibility_label }}{% endtrans %}">
+        <div class="badge teaser__badge no-padding">
         <span class="badge__container">
-          <svg class="badge__icon" aria-hidden="true">
+          <svg class="badge__icon">
```

Comment:
This SVG should remain hidden because there's an `sr-only` line right after it with more information.

Reply:
Fixed